samples/drivers/video/tcpserversink/README.rst

```diff
@@ -26,6 +29,10 @@ J35 camera connector. A USB cable should be connected from a host to the micro
 USB debug connector (J41) in order to get console output via the freelink
 interface. Ethernet cable must be connected to RJ45 connector.
 
+On :zephyr:board:`stm32n6570_dk`, the MB1854 IMX335 camera module must be plugged in
+the CSI-2 camera connector. A RJ45 ethernet cable must be plugged in the ethernet CN6
+connector.
+
```

Comment:
Probably add a note here mentioning the ISP module should be used as it cannot be added to the default west manifest.